aldryn_forms/contrib/email_notifications/cms_plugins.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from django.contrib import admin
+from django.core.mail import get_connection
+from django.utils.translation import ugettext_lazy as _
+
+from cms.plugin_pool import plugin_pool
+
+from aldryn_forms.cms_plugins import FormPlugin
+
+from .models import EmailNotification, EmailNotificationFormPlugin
+from .validators import is_valid_recipient
+
+
+logger = logging.getLogger(__name__)
+
+
+class EmailNotificationInline(admin.StackedInline):
+    model = EmailNotification
+    extra = 0
+    fieldsets = [
+        (
+            None,
+            {'fields': ['theme']}
+        ),
+        (
+            None,
+            {'fields': ['text_variables']}
+        ),
+        (
+            'Recipient',
+            {'fields': ['to_user', ('to_name', 'to_email')]}
+        ),
+        (
+            'Sender',
+            {'fields': [('from_name', 'from_email')]}
+        ),
+        (
+            'Email body',
+            {'fields': ['subject', 'body_text', 'body_html']}
+        ),
+    ]
+    readonly_fields = ['text_variables']
+    text_variables_help_text = _('variables can be used with by '
+                                 'wrapping with "${variable}" like ${variable}')
+
+    def text_variables(self, obj):
+        variables = obj.get_text_variables()
+        li_items = (u'<li>{0} | {1}</li>'.format(*var) for var in variables)
```

Comment:
This are "variables with labels" not just `variables`. Here It looks like you'd splitting variable name into single chars and providing them to the "format" function.